packages/ui/src/Button.module.scss

```diff
@@ -1,7 +1,12 @@
 @use '../styles/constants' as c;
 @use '../styles/helpers' as h;
 
+a.button {
+  text-decoration: none;
```

Comment:
What if we just add it to the `.button`? It should not hurt the HTML `button`, right?